Unity.gitignore

```diff
@@ -23,8 +23,9 @@
 # Autogenerated Jetbrains Rider plugin
 /[Aa]ssets/Plugins/Editor/JetBrains*
 
-# Visual Studio cache directory
+# Visual Studio and Visual Studio Code cache directory
 .vs/
+.vscode/
```

Comment:
It's not a cache directory, We have VS code settings and extensions recommendations commited and not ignored. I wonder what the recommandation is though.